.env.example

```
@@ -1,6 +1,10 @@
 NEXT_PUBLIC_WALLET_CONNECT_ID=12345678901234567890123456789012
 NEXT_PUBLIC_RPC_OVERRIDES='{"chain1":{"http":"https://..."}}'
 
+# Offchain fee quoting (optional)
+NEXT_PUBLIC_FEE_QUOTING_URL=       # Fee quoting service base URL
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)